doc/deployment-on-arbitrary-cluster.md

```diff
@@ -0,0 +1,30 @@
+# Deployment on Arbitrary Cluster
+
+In this guide we will cover installation of Multus and
+ovs-cni on your arbitrary cluster.
+
+# Requirements
+
+## A Cluster
+
+This guide requires you to have your own Kubernetes cluster. If you
+don't have one and just want to try Kubemacpool out, please refer to the [deployment
+on local cluster](deployment-on-local-cluster.md) guide.
+
+## Prerequisites
```

Comment:
Sounds redundant to have this under requirements and odd to have it on the same level as a cluster

Reply:
you're right. maybe it's better to join the cluster section with the introduction section. Let me know what you think